code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -103,41 +103,41 @@ \subsection{Table of Symbols}
 \\
 ${h_{z}}$ & Center of Slice Height: the distance from the lowest part of the slice to the height of the centers of slice & m
 \\
-$i$ & Index: used to show a quantity applies to only one slice & --
+$i$ & Index: representing a single slice & --
 \\
-${K_{c}}$ & Earthquake Load Factor: proportionality factor of force that weight pushes outwards; caused by seismic earth movements & --
+${K_{c}}$ & Seismic Coefficient: proportionality factor of force that weight pushes outwards; caused by seismic earth movements & --
 \\
 $M$ & Net Moment: a measure of the tendency of a body to rotate about a specific point or axis & Nm
 \\
-$N$ & Normal Force: total reactive force for a soil surface subject to a body resting on it & N
+$N$ & Normal Force: total reactive force per meter in the z-direction for a soil surface subject to a body resting on it & $\frac{\text{N}}{\text{m}}$
 \\
 $n$ & Number of Slices: the slip mass has been divided into & --
 \\
-$N'$ & Effective Normal Force: for a soil surface, subtracting pore water reactive force from total reactive force & N
+$N'$ & Effective Normal Force: per meter in the z-direction for a soil surface, subtracting pore water reactive force from total reactive force & $\frac{\text{N}}{\text{m}}$
 \\
-$P$ & Resistive Shear Force Per Meter in the Z-Direction: Mohr Coulomb frictional force that describes the limit of mobilized shear force the slice i can withstand before failure & $\frac{\text{N}}{\text{m}}$
+$P$ & Resistive Shear Force: Mohr Coulomb frictional force per meter in the z-direction that describes the limit ofmobilized shear force a slice can withstand before failure & $\frac{\text{N}}{\text{m}}$
```

Comment:
Please insert a space between "of" and "mobilized."

Reply:
Good catch, fixed in 8b2bf80af01fe221d52bd87bbb671053812b1306.